lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -190,6 +190,70 @@ export default TestModule.extend({
     };
   },
 
+  setupComponentLegacyIntegrationTest: function () {
+    var resolver = getResolver();
+    var namespace = Ember.Object.create({
+      Resolver: { create: function() { return resolver; } }
+    });
+
+    if (Ember.Application.buildRegistry) {
+      var registry;
+      registry = Ember.Application.buildRegistry(namespace);
+      registry.register('component-lookup:main', Ember.ComponentLookup);
+      this.registry = registry;
+      this.container = registry.container();
+    } else {
+      this.container = Ember.Application.buildContainer(namespace);
+      this.container.register('component-lookup:main', Ember.ComponentLookup);
+    }
+  },
+
+  setupLegacyComponentTest: function () {
```

Comment:
https://github.com/switchfly/ember-test-helpers/blob/master/lib/ember-test-helpers/test-module.js#L129-L135 determines if the container that is being used is isolated or has the full resolver (note how we clobber the underlying `resolver` [here](https://github.com/switchfly/ember-test-helpers/blob/master/lib/ember-test-helpers/test-module.js#L284-L288) after manually resolving all of the `needs` specified items).